samples/dashboards/demo/accounting/demo.js

```diff
@@ -0,0 +1,657 @@
+// eslint-disable-next-line no-underscore-dangle
+const U = Highcharts._modules['Core/Utilities.js'];
+
+const currentMonth = Date.UTC(2023, 9);
+const revTarget = 105;
+const costTarget = 89;
+
+const currentYear = new Date(currentMonth).getFullYear();
+
+const commonGaugeOptions = {
+    chart: {
+        type: 'gauge',
+        className: 'highcharts-gauge-chart',
+        marginBottom: 0
+    },
+    pane: {
+        startAngle: -90,
+        endAngle: 89.9,
+        background: null,
+        center: ['50%', '64%'],
+        size: '110%'
+    },
+    yAxis: {
+        visible: true,
+        min: 0,
+        minorTickInterval: null,
+        labels: {
+            distance: 12,
+            allowOverlap: true
+        }
+    },
+    tooltip: {
+        enabled: false
+    },
+    plotOptions: {
+        series: {
+            dial: {
+                baseWidth: 12,
+                baseLength: 0,
+                rearLength: 0
+            },
+            pivot: {
+                radius: 5
+            },
+            dataLabels: {
+                useHTML: true,
+                format: '${y}M'
+            }
+        }
+    }
+};
+
+const commonColumnOptions = {
+    accessibility: {
+        point: {
+            valuePrefix: '$'
+        }
+    },
+    chart: {
+        type: 'column',
+        className: 'highcharts-column-chart'
+    },
+    credits: {
+        enabled: false
+    },
+    xAxis: {
+        type: 'datetime',
+        min: Date.UTC(currentYear),
+        max: Date.UTC(currentYear, 11)
+    },
+    yAxis: {
+        tickInterval: 2e6
+    },
+    series: [{
+        name: 'Budget',
+        colorIndex: 1
+    }],
+    tooltip: {
+        formatter: function () {
+            const { x, y, colorIndex: color, series } = this;
+            const date = Highcharts.dateFormat('%B %Y', x);
+
+            return `<span style="font-size: 10px">${date}</span><br>
+                <span class="highcharts-color-${color}">&#9679;</span>&nbsp;
+                ${series.name}: $${(y / 1e6).toFixed(2)}M
+            `;
+        }
```

Comment:
This should be doable with a `tooltip.format`, which is preferred over callbacks because it is serializable and XSS-secure. Off the top of my head, something like this:
```js
format: `<span style="font-size: 10px">{x:%B %Y}</span><br>
                <span class="highcharts-color-{colorIndex}">&#9679;</span>&nbsp;
                {series.name}: ${(divide y 1000000):.2f}M
            `
```